CONTRIBUTING.md

```diff
@@ -37,6 +37,11 @@ You can provide the protocol compiler protoc path programmatically (only works w
 export PROTOC=$(which protoc)
 ```
 
+It is recommended to use "3.15" or newer of protoc, as some of the proto
```

Comment:
recommended or required :)

Reply:
Well...I survived without that for months ;)